packages/labs/context/src/test/late-provider_test.ts

```diff
@@ -139,4 +139,108 @@ suite('late context provider', () => {
     // `value` should now be provided
     assert.strictEqual(consumer.value, 1000);
   });
+
+  test('late element with multiple properties', async () => {
```

Comment:
This test verifies that the fix in the PR didn't break elements with multiple providers which have different callback identity but the same element.